setup_dev.py

```diff
@@ -49,7 +51,9 @@ def main():
 
     print(
         f"An editable version of pdm is installed at {pdm_path}, "
-        "you can create an alias for it for convenience.",
+        "you can create an alias for it for convenience.\n"
+        "Ensure PEP 582 is enabled by executing\n"
+        + textwrap.indent(pep582_output, " " * 4),
```

Comment:
I think this should mention evaluating the script / output of `…/pdm --pep582` in the shell instead, basically linking to https://pdm.fming.dev/#enable-pep-582-globally, and could mention the entry that should be added to PYTHONPATH here then explicitly.

What it currently outputs there is a bit long (with the fix from https://github.com/pdm-project/pdm/pull/416/commits/925b21913e5cd32b87e95c39abce1e41a8637cdd):
```
Ensure PEP 582 is enabled by executing
    if [ -n "$PYTHONPATH" ]; then
        export PYTHONPATH='/home/user/Vcs/pdm/pdm/pep582':$PYTHONPATH
    else
        export PYTHONPATH='/home/user/Vcs/pdm/pdm/pep582'
    fi
```
(see also https://github.com/pdm-project/pdm/pull/411#pullrequestreview-644641761)

Reply:
What about:

Ensure PEP 582 by adding `/home/user/Vcs/pdm/pdm/pep582` to `PYTHONPATH`.

Reply:
Sounds good in general:

Ensure PEP 582 is enabled by adding `…/pdm/pdm/pep582` to `$PYTHONPATH`, e.g. by adding the following to your shell config files:
…


It should also mention that `python -m pdm` should be used then to run PDM (either from the source checkout / root, or by adding its root also to `$PYTHONPATH` (more/too complicated then though!?)).

In general it is good to (also) describe what the shell script is meant to do.